src/Controls/src/Core/BindableObject.cs

```diff
@@ -264,17 +264,12 @@ public static void SetInheritedBindingContext(BindableObject bindable, object va
 		protected virtual void OnBindingContextChanged()
 		{
 			BindingContextChanged?.Invoke(this, EventArgs.Empty);
+
 			if (Shell.GetBackButtonBehavior(this) is BackButtonBehavior buttonBehavior)
 				SetInheritedBindingContext(buttonBehavior, BindingContext);
 
 			if (Shell.GetSearchHandler(this) is SearchHandler searchHandler)
 				SetInheritedBindingContext(searchHandler, BindingContext);
-
-			if (Shell.GetTitleView(this) is View titleView)
-				SetInheritedBindingContext(titleView, BindingContext);
-
-			if (FlyoutBase.GetContextFlyout(this) is BindableObject contextFlyout)
-				SetInheritedBindingContext(contextFlyout, BindingContext);
```

Comment:
This was not a view before and may not be part of the visual tree. Is this change expected?